draft-thomson-http-scd.md

```diff
@@ -400,10 +400,10 @@ information about their contents.  When used carefully, padding as described in
 [I-D.ietf-httpbis-encryption-encoding] can obscure the length of responses and
 reduce the information that the secondary server is able to learn.
 
-A random or unpredictable mapping from the primary resource URL on the origin to
-the URL of the content is necessary, see {{urls}}.
+A random or unpredictable mapping from the primary resource URL on the primary
+server to the URL of the content is necessary, see {{urls}}.
 
-Length hiding for header fields on responses the origin server might be more
+Length hiding for header fields on responses the primary server might be more
```

Comment:
there's a missing word in this sentence: on responses **from** the primary server